UniConnectApp/app/src/main/java/com/example/uniconnect/MainActivity.kt

```diff
@@ -57,28 +59,76 @@ class MainActivity : ComponentActivity() {
         var description by remember { mutableStateOf("")}
         //var postID by remember { mutableStateOf("")}
         val context = LocalContext.current
-        Column{
-            OutlinedTextField(
-                value = title,
-                onValueChange = {title = it},
-                label = { Text(stringResource(R.string.title))}
+        Column(
+            Modifier.fillMaxHeight(),
+            verticalArrangement = Arrangement.SpaceBetween
+        ) {
+            Text(
+                text = stringResource(id = R.string.post_title),
+                style = MaterialTheme.typography.h6,
+                modifier = Modifier.padding(start = 20.dp)
             )
```

Comment:
The Column that holds everything together has been modified to fit to the whole page. I have also changed the vertical arrangement to add space between newly added the rows.